### PR TITLE
[zh] docs: sync and update the ref

### PR DESCRIPTION
**Pull request by zhangguanzhang**
Signed-off-by: zhangguanzhang <zhangguanzhang@qq.com>
/assign @tengqm

### CODE REVIEW

File: content/zh/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 1]
```diff
@@ -30,6 +30,17 @@ It does not mean that there is a file named `kubeconfig`.
...
 {{< /note >}}
 
+<!--
+{{< warning >}}
+Only use kubeconfig files from trusted sources. Using a specially-crafted kubeconfig file could result in malicious code execution or file exposure. 
+If you must use an untrusted kubeconfig file, inspect it carefully first, much as you would a shell script.
+{{< /warning>}}
+-->
+{{< warning >}}
+只使用来自来源可靠的 kubecconfig 文件。使用特制的 kubeconfig 文件可能会导致恶意代码执行或文件暴露。
```
Comment (chenxuc) [inline content/zh/docs/concepts/configuration/organize-cluster-access-kubeconfig.md:40]: Suggest: 只使用来源可靠的 kubecconfig 文件。

Comment (zhangguanzhang) [reply content/zh/docs/concepts/configuration/organize-cluster-access-kubeconfig.md]: had done

File: content/zh/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 2]
```diff
@@ -30,6 +30,17 @@ It does not mean that there is a file named `kubeconfig`.
...
 {{< /note >}}
 
+<!--
+{{< warning >}}
+Only use kubeconfig files from trusted sources. Using a specially-crafted kubeconfig file could result in malicious code execution or file exposure. 
+If you must use an untrusted kubeconfig file, inspect it carefully first, much as you would a shell script.
+{{< /warning>}}
+-->
+{{< warning >}}
+只使用来源可靠的 kubecconfig 文件。使用特制的 kubeconfig 文件可能会导致恶意代码执行或文件暴露。
```
Comment (tengqm) [inline content/zh/docs/concepts/configuration/organize-cluster-access-kubeconfig.md:40]:
```suggestion
只使用来源可靠的 kubeconfig 文件。使用特制的 kubeconfig 文件可能会导致恶意代码执行或文件暴露。
```

**Review by tengqm — COMMENTED** [review]
Just one nit.

File: content/zh/docs/concepts/configuration/organize-cluster-access-kubeconfig.md (hunk 2 above)
Comment (zhangguanzhang) [reply content/zh/docs/concepts/configuration/organize-cluster-access-kubeconfig.md]: all had done

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**